certif/app/routes/authenticated.js

```diff
@@ -1,5 +1,12 @@
 import Route from '@ember/routing/route';
 import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+import { inject as service } from '@ember/service';
 
 export default Route.extend(AuthenticatedRouteMixin, {
+  currentCertificationCenter: service(),
+
+  model() {
+    return this.currentCertificationCenter.certificationCenter.then((certificationCenter) => certificationCenter);
```

Comment:
Heu ? Il sert à quoi le then ?

Reply:
Car : this.currentCertificationCenter.certificationCenter renvoie une promesse derrière